djangocms_url_manager/models.py

```diff
@@ -130,7 +129,9 @@ def get_content(self, show_draft_content=False):
 
 
 class UrlGrouper(AbstractUrlGrouper):
-    def get_content_queryset(self):
+    def get_content_queryset(self, show_draft_content=False):
+        if CMS_41 and hasattr(Url, "admin_manager") and show_draft_content:
```

Comment:
```suggestion
        if hasattr(Url, "admin_manager") and show_draft_content:
```
Just a suggestion: In django CMS 4 `Url` never has the attribute `admin_manager`, so the version check can go away.

Reply:
@fsbraun , You are right. I will remove this line.